test/test_dockerfile_hermetic.py

```
@@ -0,0 +1,480 @@
+        print("\n=== Cleaning up hermetic tests ===")
+
+        # Remove the Docker image
+        subprocess.run(
+            ["podman", "rmi", "-f", cls.IMAGE_NAME],
+            capture_output=True
+        )
+        print(f"✓ Image {cls.IMAGE_NAME} removed")
+
+        # Remove copied Dockerfile
+        cls._cleanup_test_env(cls.test_dir)
+        print(f"✓ Removed copied Dockerfile.hermetic")
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)